<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <!-- Launch Bounding Box Filter -->
    <include file="$(find cuboid_detection)/launch/bbox_filter.launch"/>

    <!-- Template object path for ICP -->
    <param
        name="template_cuboid_path"
        type="string"
        value="$(find cuboid_detection)/templates/template_cuboid_L200_W100_H75.pcd" />
    
    <!-- Run ICP node -->
    <node
        pkg="cuboid_detection"
        type="iterative_closest_point"
        name="iterative_closest_point"
        output="screen">
    </node>

</launch>
